internal/controller/resources_cucp.go

```diff
@@ -218,13 +218,16 @@ func (resource CuCpResources) GetDeployment(log logr.Logger, ranDeployment *work
 								corev1.EnvVar{
 									Name:  "USE_ADDITIONAL_OPTIONS",
 									Value: "--sa --log_config.global_log_options level,nocolor,time",
+									// " --gNBs.[0].E1_INTERFACE.[0].ipv4_cucp 192.168.77.2" +
```

Comment:
@jain-ashish-sam you can remove the comments, they are not relevant in this use case.